temba/channels/tests.py

```diff
@@ -7608,6 +7608,43 @@ def test_send_media(self):
             self.assertEqual(mock.call_args[0][1]['caption'], "Test message")
             self.assertEqual(mock.call_args[0][1]['chat_id'], "1234")
 
+    def test_send_quick_replies(self):
+        metadata = """
+        {
+            "quick_replies": [
+                {
+                    "payload": "yes",
+                    "title": "Yes"
```

Comment:
Aren't these supposed to be a single value now?

Reply:
Yes, I'll change the tests after reviews :)